pkg/scheduler/framework/types_test.go

```diff
@@ -1623,6 +1623,8 @@ func TestPodInfoCalculateResources(t *testing.T) {
 
 func TestCalculatePodResourcesWithResize(t *testing.T) {
 	featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.InPlacePodVerticalScaling, true)
+	featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.InPlacePodLevelResourcesVerticalScaling, true)
```

Comment:
nit: Use the bulk method
```suggestion
	featuregatetesting.SetFeatureGatesDuringTest(t, utilfeature.DefaultFeatureGate, featruegatetesting.FeatureOverrides{
		features.InPlacePodVerticalScaling: true,
		features.InPlacePodLevelResourcesVerticalScaling: true,
	})
```